docs/filter-api.md

```diff
@@ -19,4 +19,44 @@ dust.filters.myFilter = function(value) {
 ``` 
 ###Parameters
 * value: The value that is used in the dust reference.  This is the left hand argument in the filter call above.
-* @return {String} The modified value
\ No newline at end of file
+* @return {String} The modified value
+
+###Example
+
+In this example, we will define an javascript filter, which takes a string and returns another string that is safe to use in javascript by escaping special javascript characters.
+
+#### Javascript definition of a JSON filter
+```
+var BS = /\\/g,
+    FS = /\//g,
+    CR = /\r/g,
+    LS = /\u2028/g,
```

Comment:
What in the world are these things? Why don't you make the example a much simpler filter.